bundles/org.openhab.binding.nanoleaf/README.md

```diff
@@ -272,7 +274,8 @@ end
 
 rule "Nanoleaf canvas touch detection Panel 2"
 when
-    Item Panel2SingleTap changed from OFF to ON
+    Item Panel2SingleTap changed from OFF to ON or
+    Item Panel2SingleTap changed from NULL to ON or
```

Comment:
```suggestion
    Item Panel2SingleTap changed from NULL to ON
```

Reply:
Not sure if I understand the change. Do you recommend to go from 

```
    Item Panel1DoubleTap changed from OFF to ON or
    Item Panel1DoubleTap changed from NULL to ON
```

to

```    
    Item Panel1DoubleTap changed from NULL to ON or
    Item Panel1DoubleTap changed from OFF to ON 
```

i.e. you switched the lines?

cheers
Stefan

Reply:
I removed the `or` at the end of the second line

Reply:
Thanks, I overlooked that.